src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,48 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
+
+  static get alive() {
+    return Animal._alive;
+  }
+
+  static set alive(aliveArray) {
+    Animal._alive = aliveArray;
+  }
```

Comment:
There is a conflict with the static property 'alive' and the static getter/setter of the same name. This will lead to issues when trying to access or modify the 'alive' array. Consider renaming the getter/setter to avoid this conflict.